### PR TITLE
Improve readme

### PR DESCRIPTION
**Pull request by aareet**
### Description
Improves README with additional information.

### Acceptance tests
- [ ] Have you added an acceptance test for the functionality being added?
- [ ] Have you run the acceptance tests on this branch? (If so, please include the test log in a gist)

### CODE REVIEW

File: _about/FAQ.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+The HashiCorp Terraform Kubernetes provider team is :
+
+* Phil Sautter, Product Manager - [@redeux](https://github.com/redeux)
+* Alex Somesan, Engineer - [@alexsomesan](https://github.com/alexsomesan)
+* John Houston, Engineer - [@jrhouston](https://github.com/jrhouston)
+* Stef Forrester, Engineer - [@dak1n1](https://github.com/dak1n1)
+* Aareet Shermon, Engineering Manager - [@aareet](https://github.com/aareet)
+
+Our collaborators are:
+* Maximo Cuadros - [@mcuadros](https://github.com/mcuadros)
```
Comment (aareet) [inline _about/FAQ.md:14]: @meyskens @mcuadros please let me know if you are comfortable being included in this list.

Comment (meyskens) [reply _about/FAQ.md:14]: I am no longer actively contributing here any longer as I changed employers

Comment (mcuadros) [reply _about/FAQ.md:14]: From my side of course.

Comment (aareet) [reply _about/FAQ.md:14]: No problem, thank you for all you've done so far. I'll update the list

**Pull request merged**